src/main/java/com/czertainly/core/service/scep/impl/ScepServiceImpl.java

```diff
@@ -505,7 +504,7 @@ private ScepResponse generateCsr(ScepRequest scepRequest, IntuneScepServiceClien
         ClientCertificateRequestDto requestDto = new ClientCertificateRequestDto();
         if (raProfile != null) requestDto.setRaProfileUuid(raProfile.getUuid());
         try {
-            requestDto.setPkcs10(new String(Base64.getEncoder().encode(scepRequest.getPkcs10Request().getEncoded())));
+            requestDto.setRequest(new String(Base64.getEncoder().encode(scepRequest.getPkcs10Request().getEncoded())));
```

Comment:
Is this working as expected when the builder for `ClientCertificateRequestDto` is not used?